packages/client/package.json

```diff
@@ -26,19 +26,13 @@
       "import": "./dist/index.mjs",
       "require": "./dist/index.js",
       "default": "./dist/index.js"
-    },
-    "./observable": {
-      "import": "./dist/observable/index.mjs",
-      "require": "./dist/observable/index.js",
-      "default": "./dist/observable/index.js"
```

Comment:
Tree shaking should be fine in that case. I think importing from `@trpc/server` makes more sense because it's being used server-side to build the router.